### PR TITLE
fix: fix kwargs in .fill() for WeightedMean&Mean storage

### PR DESCRIPTION
**Pull request by LovelyBuggies**
@henryiii Fix the error of "axis name could not be found" in

```python
h = NamedHist.new.Reg(10, 0, 1, name="x").WeightedMean().fill(x=[0.5, 0.5], weight=[1, 1], sample=[1, 1])
```

and allow TestClass to work (update `.new` proxy tests).

### CODE REVIEW

File: src/hist/quick_construct.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def Double(self) -> "BaseHist":
     def Int64(self) -> "BaseHist":
         return self.hist_class(*self.axes, storage=storage.Int64())
 
-    def AtomicInt64(self) -> "BaseHist":
+    def AutomicInt64(self) -> "BaseHist":
```
Comment (henryiii) [inline src/hist/quick_construct.py:106]:
```suggestion
    def AtomicInt64(self) -> "BaseHist":
```

File: tests/test_base.py [hunk 2]
```diff
@@ -691,40 +691,39 @@ def test_base_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
-class test_base_storage_proxy:
+class Test_base_storage_proxy:
```
Comment (henryiii) [inline tests/test_base.py:694]:
```suggestion
class TestBaseStorageProxy:
```
Classes should always be CamelCase. :)

File: tests/test_general.py [hunk 3]
```diff
@@ -697,40 +697,39 @@ def test_general_index_access():
     plt.close("all")
 
 
-class test_general_storage_proxy:
+class Test_general_storage_proxy:
```
Comment (henryiii) [inline tests/test_general.py:700]:
```suggestion
class TestGeneralStorageProxy:
```

File: tests/test_named.py [hunk 4]
```diff
@@ -438,40 +438,39 @@ def test_named_access():
         h[0j, -0j + 2, "hi", True]
 
 
-class test_named_storage_proxy:
+class Test_named_storage_proxy:
```
Comment (henryiii) [inline tests/test_named.py:441]:
```suggestion
class TestNamedStorageProxy:
```

File: src/hist/quick_construct.py (hunk 1 above)
Comment (LovelyBuggies) [reply src/hist/quick_construct.py]: Okay, sorry for the spelling.

**Comment by LovelyBuggies** [comment]
@henryiii Your commit doesn't cover some places, I made a new one.

**Pull request merged**